pkg/cvo/metrics.go

```diff
@@ -114,6 +119,10 @@ penultimate completed version for 'completed'.
 			Name: "cluster_version_operator_update_retrieval_timestamp_seconds",
 			Help: "Reports when updates were last successfully retrieved.",
 		}, []string{"name"}),
+		clusterVersionConditionalUpdateConditionSeconds: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "cluster_version_conditional_update_condition_seconds",
+			Help: "Reports when condition on conditional updates were last updated",
```

Comment:
nit: this doesn't clarify the "number of seconds before now, modulo serve/scrape latencies" vs. "number of seconds after the Unix epoch" distinction.  Since you're currently going with the former, is it worth calling that out with some words here?  I'm agnostic about whether you go into the serving/scraping latencies or not, since those will always be well short of `CannotEvaluateConditionalUpdates`'s hour threshold in any remotely healthy system.

Reply:
I'm not going into the scraping/serving latencies space (but at least I now understand better why some metrics are like that) in the documentation, but I improved the message to say better what is the meaning of the number.